package.json

```diff
@@ -55,10 +55,10 @@
     "express-session": "1.15.3",
     "express-winston": "2.4.0",
     "node-w3capi": "1.4.0",
-    "nodemailer-mock-transport": "1.3.0",
     "nodemailer": "4.0.1",
+    "nodemailer-mock-transport": "1.3.0",
     "object-assign": "4.1.1",
-    "octokat": "0.4.18",
+    "octokat": "^0.10.0",
```

Comment:
Better to set specific versions, as with all other dependencies (ie, remove caret)?